src/core/dataframe.js

```diff
@@ -65,22 +65,45 @@ export default class Dataframe {
         this.style = style;
```

Comment:
I think this `setStyle` function can be removed

Reply:
Yes, that's legacy code

Reply:
Fixed